src/main/java/com/timgroup/statsd/Telemetry.java

```diff
@@ -0,0 +1,150 @@
+package com.timgroup.statsd;
+
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Telemetry {
+
+    public static int DEFAULT_FLUSH_INTERVAL = 10000; // 10s
+
+    protected AtomicInteger metricsSent;
+    protected AtomicInteger eventsSent;
+    protected AtomicInteger serviceChecksSent;
+    protected AtomicInteger bytesSent;
+    protected AtomicInteger bytesDropped;
+    protected AtomicInteger packetsSent;
+    protected AtomicInteger packetsDropped;
+    protected AtomicInteger packetsDroppedQueue;
+
+    protected String metricsSentMetric;
```

Comment:
This is going to conflict a little bit with the implementation in #105, no worries we can merge this first and I can fix it in that PR. These should be `final` too, I can also fix that in a follow-up.